application/management/commands/load_csv.py

```diff
@@ -14,7 +15,7 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         data_filename = options['data_file_csv']
 
-        reverse_country_list = {name:code for code, name in countries}
+        reverse_country_list = {name:code for code, name in countries} 
```

Comment:
Please remove the white space at the end of the line (there are some extensions that will do this for you, in most code editing software)